var/spack/repos/builtin/packages/r-spades-tools/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RSpadesTools(RPackage):
+    """Tools for Spatially Explicit Discrete Event Simulation (SpaDES) Models
+
+    Provides GIS and map utilities, plus additional modeling tools for developing
+    cellular automata, dynamic raster models, and agent based models in 'SpaDES'.
+    Included are various methods for spatial spreading, spatial agents, GIS
+    operations, random map generation, and others. See '?SpaDES.tools' for an
+    categorized overview of these additional tools."""
+
+    homepage = "https://spades-tools.predictiveecology.org/"
+    url      = "https://cloud.r-project.org/src/contrib/SpaDES.tools_0.3.6.tar.gz"
+    list_url = "https://cloud.r-project.org/src/contrib/Archive/SpaDES.tools"
+
+    maintainers = ['dorton21']
+
+    version('0.3.6', sha256='661f8ee792874e7447be78103775b63f18ec69e773a7b275dd977adb406dd3e5')
+
+    depends_on('r@3.5.0:', type=('build', 'run'))
+    depends_on('r-backports', type=('build', 'run'))
+    depends_on('r-checkmate@1.8.2:', type=('build', 'run'))
+    depends_on('r-circstats@0.2-4:', type=('build', 'run'))
```

Comment:
This package needs a package for `r-circstats`. The `r-circstat` package in #21777 was supposed to be that package, but was misnamed. The best course is to rename that package with new PR.